src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -318,13 +318,11 @@ class FrontendGenImpl {
     // attributes.push_back(dilations_attr)
     // similar situation for pads, strides in AveragePool
     // axes of ReduceSum,  pads, strides, dilations and kernel_shape of MaxPool
-    // TODO: fix this after type inference
-    int nOps = node.input().size();
 
-    if (nOps == 2)
-      buildOperation<mlir::ONNXConvNoBiasOp>(node, nOps, nOut);
-    else
-      buildOperation<mlir::ONNXConvOp>(node, nOps, nOut);
+    // nIn is set to 3, and node may have only 2 input when bias not set. 
+    // buildOpertion will automatically add "null" typed input operands to
+    // match nIn value.
+    buildOperation<mlir::ONNXConvOp>(node, nIn, nOut);
   }
```

Comment:
I feel that the ImportNodeConv is not needed any more.
Accordingly, "conv" should be removed from special_op_handler dictionary in gen_doc.py

Reply:
Tong, good suggestion, done.

I will create a separate issue suggesting that we build the inc files each build, as it is hard to check that we have no issue manually changing the gen_doc.py